Dockerfile

```diff
@@ -71,4 +71,4 @@ RUN mkdir /bzt-configs \
   && rm -rf /tmp/*
 
 WORKDIR /bzt-configs
-CMD bzt -l /tmp/artifacts/bzt.log /bzt-configs/*.yml
+ENTRYPOINT ["sh", "-c", "bzt /bzt-configs/*.yml -l /tmp/artifacts/bzt.log \"$@\"", "ignored"]
```

Comment:
Man, you are cool. You making something I never had time to do.

If we migrate to ENTRYPOINT, let's get rid of *.yml on docker side. I did not like that solution from the beginning, just did not know about ENTRYPOINT at that moment.

Getting rid of *.yml command also simplifies mounting part of Docker invocation and makes usage pattern more natural. Don't forget to reflect that in doc, also as part of this PR.

Good stuff man, good stuff!